src/generator/factory.ts

```diff
@@ -2,7 +2,7 @@ import { faker } from '@faker-js/faker'
 import { getRandomItemFromArray } from '../utils/data-helpers'
 import { Entities } from './entities'
 import { Types } from './types'
-
+// TODO: add parameters to modify client data
```

Comment:
Good point. Should add modification to overwrite generator and control it form test level on demand